internal/github/utils.go

```diff
@@ -0,0 +1,25 @@
+package github
+
+import (
+	"net/http"
+
+	gh "github.com/google/go-github/v33/github"
+)
+
+// parseBoolResponse determines the boolean result from a GitHub API response.
+// Several GitHub API methods return boolean responses indicated by the HTTP
+// status code in the response (true indicated by a 204, false indicated by a
```

Comment:
Why 204 status means true?

Reply:
It does not refer to Github Get methods. Only bool responses return 204 or 404. For example [this](https://docs.github.com/en/rest/reference/repos#check-if-vulnerability-alerts-are-enabled-for-a-repository) or [this](https://docs.github.com/en/rest/reference/repos#check-if-a-user-is-a-repository-collaborator).
In case of get method it returns 200 or 404.
Corrected the function comment.